bokeh/resources.py

```diff
@@ -274,6 +274,25 @@ def _file_paths(self, kind):
         paths = [ join(bokehjs_dir, kind, file) for file in files ]
         return paths
 
+    def _collect_external_resources(self, resource_attr):
+        """ Collect external resources set on resource_attr attribute (expected
+        to be "__external_js_resources__" or "__external_css_resources__") of models.
```

Comment:
Although this is an internal method, use appropriate syntax (double backticks) to indicate inline code for the sake of consistency.